src/datepicker/docs/readme.md

```diff
@@ -1,145 +1,150 @@
-
- * `starting-day`
- 	_(Defaults: 0)_ :
- 	Starting day of the week from 0-6 (0=Sunday, ..., 6=Saturday).
-
- * `init-date`
- 	:
- 	The initial date view when no model value is specified.
-
- * `min-mode`
-   _(Defaults: 'day')_ :
-   Set a lower limit for mode.
-
- * `max-mode`
-   _(Defaults: 'year')_ :
-   Set an upper limit for mode.
-
- * `format-day`
- 	_(Default: 'dd')_ :
- 	Format of day in month.
-
- * `format-month`
- 	_(Default: 'MMMM')_ :
- 	Format of month in year.
-
- * `format-year`
- 	_(Default: 'yyyy')_ :
- 	Format of year in year range.
-
- * `format-day-header`
- 	_(Default: 'EEE')_ :
- 	Format of day in week header.
-
- * `format-day-title`
- 	_(Default: 'MMMM yyyy')_ :
- 	Format of title when selecting day.
-
- * `format-month-title`
- 	_(Default: 'yyyy')_ :
- 	Format of title when selecting month.
-
- * `year-range`
- 	_(Default: 20)_ :
- 	Number of years displayed in year selection.
-
- * `shortcut-propagation`
-  _(Default: false)_ :
+Our datepicker is flexible and fully customizable.
+
+You can navigate through days, months and years.
+
+It comes in two formats, a inline `uib-datepicker` and a `uib-datepicker-popup` to be embedded in an input.
+
+The datepicker has 3 modes:
+
+* `day` - In this mode you're presented with a 6-week calendar for an specified month.
+* `month` - In this mode you can select a month within a selected year.
+* `year` - In this mode you are presented with a range of years (20 by default).
+
+### uib-datepicker settings ###
+
+* `custom-class (date, mode)`
+  _(Default: `null`)_ -
+  An optional expression to add classes based on passing date and current mode.
+  
+* `date-disabled (date, mode)`
+  _(Default: `null`)_ -
+  An optional expression to disable visible options based on passing date and current mode.
+
+* `datepicker-mode`
+  <i class="glyphicon glyphicon-eye-open"></i>
+  _(Default: `day`)_ -
+  Current mode of the datepicker _(day|month|year)_. Can be used to initialize datepicker in a specific mode.
+
+* `format-day`
+  _(Default: `dd`)_ -
+  Format of day in month.
+
+* `format-month`
+  _(Default: `MMMM`)_ -
+  Format of month in year.
+
+* `format-year`
+  _(Default: `yyyy`)_ -
+  Format of year in year range.
+
+* `format-day-header`
+  _(Default: `EEE`)_ -
+  Format of day in week header.
+
+* `format-day-title`
+  _(Default: `MMMM yyyy`)_ -
+  Format of title when selecting day.
+
+* `format-month-title`
+  _(Default: `yyyy`)_ -
+  Format of title when selecting month.  
+
+* `init-date`
+  _(Default: `null`)_ -
+  The initial date view when no model value is specified.
+  
+* `max-date`
+  <i class="glyphicon glyphicon-eye-open"></i>
+  _(Default: `null`)_ -
+  Defines the maximum available date.
+  
+* `max-mode`
+  _(Default: `year`)_ -
+  Set an upper limit for mode.
+  
+* `min-date`
+  <i class="glyphicon glyphicon-eye-open"></i>
+  _(Default: `null`)_ -
+  Defines the minimum available date.
+
+* `min-mode`
+  _(Default: `day`)_ -
+  Set a lower limit for mode.
```

Comment:
Same.  The only reason is to be consistent since we're using "Defines" as well...